pkg/virt-controller/services/template.go

```diff
@@ -282,7 +298,15 @@ func (t *templateService) RenderMigrationManifest(vmi *v1.VirtualMachineInstance
 }
 
 func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (*k8sv1.Pod, error) {
-	return t.renderLaunchManifest(vmi, nil, false)
+	backendStoragePVCName := ""
+	if backendstorage.IsBackendStorageNeededForVMI(&vmi.Spec) {
+		backendStoragePVC := backendstorage.PVCForVMI(t.persistentVolumeClaimStore, vmi)
+		backendStoragePVCName = backendStoragePVC.Name
```

Comment:
This is a segfault, if there's no pvc yet this will panic, you have to check the pvc pointer instead of the name being empty.